components/text-control/index.js

```diff
@@ -5,7 +5,7 @@ import BaseControl from '../base-control';
 import withInstanceId from '../higher-order/with-instance-id';
 import './style.scss';
 
-function TextControl( { label, value, help, className, instanceId, onChange, type = 'text', ...props } ) {
+function TextControl( { label, value, help, className, instanceId, min, onChange, type = 'text', ...props } ) {
```

Comment:
No need for the additional `min` attribute as `...props` will pass the additional props added to the component

Reply:
Whoops, sorry, didn't realize that 😳 Just pushed [a new commit](https://github.com/WordPress/gutenberg/pull/7656/commits/5a48bc2f9b9c8a3f0df245ec4d0cb6c299853a6b) which removes any changes from the component file.

Thank you so much for the review @ajitbohra ❤️